resources/lang/en/labels.php

```diff
@@ -173,6 +173,7 @@
         'singular' => 'Transaction',
         'plural' => 'Transactions',
         'add_button' => 'Add Transaction',
+        'export_button' => 'Export Transaction',
```

Comment:
Please add english string to all languages

Reply:
I have just added more label translation